include/ck/tensor_operation/gpu/block/reduction_functions_blockwise.hpp

```diff
@@ -36,6 +36,13 @@
 
 namespace ck {
 
+// clang-format off
+// Assume:
+//  1) work_buffer is LDS allocated outside as workspace, does not include any in/out data
```

Comment:
work_buffer may or may not be allocated on LDS, it's up to the user of this function to decide

Reply:
Agree!  Even though currently it could only be LDS.